### ISSUE TITLE
MAX_MINIMUM_VOTE_THRESHOLD is unnecessarily low

### ISSUE THREAD

**Issue by NCGThompson** [BrightSpots/rcv#563]
Currently MAX_MINIMUM_VOTE_THRESHOLD is set to 1000000, but it is plausible that larger jurisdictions may want more than that. As far as I know, the maximum is just a sanity check for validation and there is no reason to keep it that low.

A better value would be on the order of the amount of voters this software can accommodate, or the number of humans on Earth. 1000000000 is probably good.

**Comment by tarheel** [comment]
I agree there's no technical reason that this couldn't be higher. That said, the likelihood of a real election wanting to set a minimum higher than 1 million seems very low, at least in the foreseeable future.
